### PR TITLE
Fix the top layers in inception_v3 model

### PR DESCRIPTION
**Pull request by parkerzf**
According to https://github.com/tensorflow/models/blob/master/slim/nets/inception_v3.py,
1) The stride of avg_pool is 1.
2) There is a dropout layer between avg_pool and flatten.

### CODE REVIEW

**Comment by fchollet** [comment]
Closing outdated PR. If you still care about the content of the PR, please submit a new PR to master, updated for the Keras 2.0 API.

**Comment by ozabluda** [comment]
This was fixed in https://github.com/fchollet/keras/commit/f516cc6ddefeeecec0e42e1f25deee24737af2bc. inception_v3.py was moved to https://github.com/tensorflow/models/blob/master/research/inception/inception/slim/inception_model.py#L320
